src/monitors/highlight.js

```diff
@@ -62,7 +62,7 @@ module.exports = class extends Monitor {
 		if (!msg.guild.members.has(_member.id)) return;
 		if (msg.mentions.users.has(_member.id)) return;
 		await _member.settings.sync(true);
-		if (_member.settings.blacklistedUsers.includes(msg.author.id) || _member.settings.blacklistedChannels.includes(msg.channel.id) || _member.id === msg.author.id) return;
+		if (_member.settings.blacklistedUsers.includes(msg.author.id) || _member.settings.blacklistedChannels.includes(msg.channel.id) || _member.user == msg.author) return;

```

Comment:
```suggestion
		if (_member.settings.blacklistedUsers.includes(msg.author.id) || _member.settings.blacklistedChannels.includes(msg.channel.id) || _member.user === msg.author) return;
```
Or keep the ID one, either works.

Reply:
lol, sorry it just bugged me.